src/typeahead/typeahead.js

```diff
@@ -351,6 +362,9 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.debounce', 'ui.bootstrap
       var model, item;
 
       selected = true;
+      //indicator that some choise selected for clear button
```

Comment:
a couple things:
* prefer end-of-line comments as opposed to comments that take their own line. _especially_ when short
* it should be "choice"
* please put a space between the `//` and the first character of the comment
* remove the blank line between this variable declaration and the line for `locals[parserResult...`
* please pick a better variable name than `selectedM` - variable names should describe what it is.